.github/workflows/python-ci.yml

```diff
@@ -49,5 +49,7 @@ jobs:
       run: make install-dependencies
     - name: Linters
       run: make lint
+    - name: Check MkDocs
```

Comment:
thanks for the suggestion!

Reply:
we need to run this for every change, since comment change can cause mkdocs strict mode to error, see https://github.com/apache/iceberg-python/pull/1334/files#diff-23e8153e0fd497a9212215bd2067068f3b56fa071770c7ef326db3d3d03cee9bL825

Reply:
Ah, I see. That makes a lot of sense 👍